packages/server/tests/ListUserFollowers.test.ts

```diff
@@ -0,0 +1,385 @@
+                    }),
+                ),
+                'nickname',
+            );
+        ///
+
+        const { body: page1BodyRaw } = await supertest(BASE_URL)
+            .post(urlcat(TEST_USER_ROUTES_GROUP_PREFIX, 'search/followers'))
+            .send({
+                page: 1,
+                searchQuery: '',
+                userID: searchedUserID,
+                tmpAuthUserID: requestingUserID,
+            } as ListUserFollowersRequestBody)
+            .expect('Content-Type', /json/)
+            .expect(200);
+        const page1BodyParsed =
+            ListUserFollowersResponseBody.parse(page1BodyRaw);
+        console.log({ pageBodyParsed: page1BodyParsed });
+        assert.equal(page1BodyParsed.page, 1);
+        assert.equal(
+            page1BodyParsed.totalEntries,
+            sortedByNicknameSearchedUserFollowersUserSummary.length,
+        );
+        assert.isTrue(page1BodyParsed.hasMore);
+        assert.equal(page1BodyParsed.data.length, PAGE_MAX_LENGTH);
+        assert.deepEqual(
+            page1BodyParsed.data,
+            sortedByNicknameSearchedUserFollowersUserSummary.slice(
+                0,
+                PAGE_MAX_LENGTH,
+            ),
+        );
+
+        const { body: page2BodyRaw } = await supertest(BASE_URL)
+            .post(urlcat(TEST_USER_ROUTES_GROUP_PREFIX, 'search/followers'))
+            .send({
+                page: 2,
+                searchQuery: '',
+                userID: searchedUserID,
+                tmpAuthUserID: requestingUserID,
+            } as ListUserFollowersRequestBody)
+            .expect('Content-Type', /json/)
+            .expect(200);
+        const page2BodyParsed =
+            ListUserFollowersResponseBody.parse(page2BodyRaw);
+        console.log({ pageBodyParsed: page2BodyParsed });
+        assert.equal(page2BodyParsed.page, 2);
+        assert.equal(
+            page2BodyParsed.totalEntries,
+            sortedByNicknameSearchedUserFollowersUserSummary.length,
+        );
+        assert.isFalse(page2BodyParsed.hasMore);
+        assert.equal(
+            page2BodyParsed.data.length,
+            sortedByNicknameSearchedUserFollowersUserSummary.length -
+                PAGE_MAX_LENGTH,
+        );
+        assert.deepEqual(
+            page2BodyParsed.data,
+            sortedByNicknameSearchedUserFollowersUserSummary.slice(
+                PAGE_MAX_LENGTH,
+                PAGE_MAX_LENGTH * 2,
+            ),
+        );
+    });
+
+    test('It should retrieve filtered user followers', async (assert) => {
+        const requestingUserID = datatype.uuid();
+        const searchedUserID = datatype.uuid();
+        const searchedUser = await User.create({
+            uuid: searchedUserID,
+            nickname: internet.userName(),
+        });
+        await User.create({
+            uuid: requestingUserID,
+            nickname: internet.userName(),
+        });
+
+        const searchQuery = datatype.string(1);
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Nice!